src/main/java/edu/ucla/library/libcal/Config.java

```diff
@@ -36,11 +50,44 @@ public final class Config {
      */
     public static final String LIBCAL_BASE_URL = "LIBCAL_BASE_URL";
 
+    /**
+     * The optional configuration property for the number of authentication retry attempts to make.
+     */
+    public static final String LIBCAL_AUTHENTICATION_RETRY_COUNT = "LIBCAL_AUTHENTICATION_RETRY_COUNT";
+
+    /**
+     * The optional configuration property for the number of seconds to wait between authentication retry attempts.
+     */
+    public static final String LIBCAL_AUTHENTICATION_RETRY_DELAY = "LIBCAL_AUTHENTICATION_RETRY_DELAY";
+
+    /**
+     * The optional configuration property for the number of seconds before OAuth token expiration to attempt a refresh.
+     */
+    @SuppressWarnings({ "PMD.LongVariable" })
```

Comment:
Re: LIBCAL_AUTHENTICATION_EXPIRES_IN_PADDING why not LIBCAL_AUTH_EXPIRES_IN_PADDING (and then mirroring the shorter name in the other variables too)? This seems like a variable name that can be shortened without losing any meaning.

Reply:
Agreed, done